Plugins~/Src/MeshSyncClientBlender/python/3.3.0/unity_mesh_sync.py

```diff
@@ -25,6 +25,8 @@
 # Called after export is finished
 mesh_sync_on_post_export = []
 
+# Called when the server requests a python callback
+mesh_sync_server_requested_callback = []
```

Comment:
how is this array populated?

Reply:
This can be populated by other python code.